src/run/TroubleshootAnalyses.ts

```diff
@@ -0,0 +1,164 @@
+import { writeFileSync } from "fs";
+import { simpleColouriser } from "../helpers/colouriser.js";
+import Config from "../model/Config.js";
+import { Inventory } from "../state/inventory.js";
+import { StateManager } from "../state/state-manager.js";
+import { Logger } from "./logs/logs-handler.js";
+import analysisTasks from "./troubleshoot/analysis/analysis-tasks.js";
+import AnalysisTask, { AnalysisFailureLevel, AnalysisIssue, AnalysisTaskOutcome } from "./troubleshoot/analysis/AnalysisTask.js";
```

Comment:
They seem all typescript files, so do we need to import "js" ?
Just commenting on this file, but the same doubt stands for the other files of this PR too

Reply:
I've responded in slack but incase this is of interest to anyone else viewing the PR not part of our slack thread.

Essentially this comes down to the Typescript configuration settings around targets/module resolution. I couldn't find anything other than this which would work with the `oclif` library when I set up originally. Maybe with someone with better typescript knowledge may be able to correct this so this is not needed.